src/EventListener/FormListener.php

```diff
@@ -0,0 +1,83 @@
+<?php
+namespace Bolt\EventListener;
+
+use Bolt\Config;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
+use Symfony\Component\HttpFoundation\Session\Storage\Handler\NativeFileSessionHandler;
+use Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage;
+
+/**
+ * Symfony Forms listener.
```

Comment:
I'm lost as to what this class is doing. If session isn't started, it's creating a separate one? Where does `$this->session` get used?